unfetter-discover/unfetter-ui
# focus first question

fixes unfetter-discover/unfetter#807


## To Test
* pull branch
* visit assessments wizard or creation page
* every time you click next or the side nav, when the questions refresh, ensure the first one is focused

Note: I think the first load of the first page, might not focus correctly, but all other pages work
